src-tauri/src/main.rs

```diff
@@ -157,6 +157,10 @@ fn main() {
         };
         set_is_accessory_policy(false);
         std::thread::sleep(std::time::Duration::from_millis(5));
+        #[cfg(target_os = "windows")]
```

Comment:
```suggestion
        #[cfg(not(target_os = "macos"))]
```

I'm guessing it might be needed on Linux as well